test/binCases/config-location/find-recursively/packages/foo/find-recursively.test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
```

Comment:
This test is not clear to me. Why test itself is nested while the configuration file is at top level? Shouldn't be the other way around?

```
- testfile.test.js
--- folder
-------- folder
-------------- webpack.config.js
-------------- src/index.js (this is 0CJS)
```

Reply:
Maybe the title of the PR is misleading but in the issue, the `package.json` (like the test file here) is nested and the `webpack.config.js` is on the top level. It should work like `git config` detection. Do I misunderstand the issue? (https://github.com/webpack/webpack-cli/issues/689#issue-380157084)

Reply:
My mistake! I misunderstood the requirements! Now it's more clear :)